lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
what do you think about moving this into its own script (report-startup or whatever) and included in `ReportGenerator.getReportJS` if `reportContext` is cli or extension? That seems like it may be clearer it's just for that context, but I'm not sure